yogstation/code/modules/jobs/job_types/captain.dm

```diff
@@ -1,3 +1,7 @@
 /datum/job/captain
 	exp_requirements = 300
-	exp_type_department = EXP_TYPE_COMMAND
\ No newline at end of file
+	exp_type_department = EXP_TYPE_COMMAND
+
+/datum/job/hop/New()
+	minimal_access += ACCESS_CAPTAIN
```

Comment:
**EXPECTATION**
adding captain to heads during lowpop rounds
**REALITY**
adding captain to heads during highpop rounds

Reply:
I hate the way everything works sometimes